Ginger/Ginger/Actions/ActionConversion/ConversionStatusReportPage.xaml.cs

```diff
@@ -219,9 +220,16 @@ private async void SaveButtonClicked(object sender, RoutedEventArgs e)
                     {
                         if (bf.IsSelected)
                         {
-                            bf.SaveStatus = eConversionSaveStatus.Saving;
-                            WorkSpace.Instance.SolutionRepository.SaveRepositoryItem(bf.BusinessFlow);
-                            bf.SaveStatus = eConversionSaveStatus.Saved;
+                            if (bf.ConvertedActionsCount > 0)
+                            {
+                                bf.SaveStatus = eConversionSaveStatus.Saving;
+                                WorkSpace.Instance.SolutionRepository.SaveRepositoryItem(bf.BusinessFlow);
+                                bf.SaveStatus = eConversionSaveStatus.Saved; 
+                            }
+                            else
+                            {
+                                bf.SaveStatus = eConversionSaveStatus.NA;
```

Comment:
Yes, the count column is updated at run time the number of action converted, we have the action to be converted but if by mistake any action is not converted (due to some error) then it will not save